config/sync/paragraphs.paragraphs_type.stanford_layout.yml

```diff
@@ -16,5 +16,6 @@ behavior_plugins:
     enabled: true
     available_layouts:
       layout_paragraphs_1_column: '1 Column'
+      layout_paragraphs_1_column_filters: '1 Column with Filters'
```

Comment:
How about 1 Column for Filtered Lists?  Or  1 Column (Wide)?  "With" makes it sound like the filters might appear. And they won't! Trying to think of other reasons we might need it, too.

Reply:
Hi @buttonwillowsix Ok. I made a new pr for the new layout names and the config changes.  
https://github.com/SU-SWS/stanford_profile/pull/888
https://github.com/SU-SWS/stanford_profile_helper/pull/384
Are they pointed to the right feature branch?